miasm2/expression/expression.py

```diff
@@ -151,7 +151,10 @@ def get_r(self, mem_read=False, cst_read=False):
 
     def get_w(self):
         return self.arg.get_w()
-
+    
+    def is_function_call(self):
```

Comment:
This code snippet seems to be specific to `ExprAff` and should not be implemented in `Expr` (`.src` undefined).
If this code can be called on any `Expr`, you could rewrite it as follow:
```Python
class Expr():
    ...
    def is_function_call(self):
        "Some Docstring"
        return False
    ...
...
class ExprAff(Expr):
    ...
    def is_function_call(self):
        return isinstance(self.src, ExprOp) and self.src.op.startswith('call')
...
```